feed/tests/test_post_model.py

```diff
@@ -0,0 +1,85 @@
+import pytest
+from feed.models import Post
+import django.contrib.auth
+
+
+User = django.contrib.auth.get_user_model()
+POST_TITLE = "Some First Post Title"
+POST_CONTENT = "Some first post content here"
+
+
+@pytest.fixture
+def users(db):
```

Comment:
You can use random user creation here with random passwords
each time fixture is called you have user, and users are returned

Reply:
Do you mean to do so by using additional package like this [link](https://eshlox.net/2019/06/11/create-random-data-for-your-django-unit-tests-using-model-mommy)?

Is it necessary?

Reply:
consider this as an idea
```
import random
import string

print ( 'user_' + ''.join(random.choice(letters) for i in range(2)))
```

Reply:
Done and added to all test files, will move this and common fixtures to a single file in a future PR